src/integration-tests/integration-test.py

```diff
@@ -178,6 +178,18 @@ class Integration(BaseModel):
         },
         use_setup=True,
     ),
+    "windows-libfuzzer-load-library": Integration(
+        template=TemplateType.libfuzzer,
+        os=OS.windows,
+        target_exe="fuzz.exe",
+        inputs="seeds",
+        wait_for_files={
+            ContainerType.inputs: 2,
+            ContainerType.unique_reports: 1,
+            ContainerType.coverage: 1,
```

Comment:
We wait for files to appear, but currently have no ability to download/make assertions about them, right?

Reply:
Yep.